js/spCreateTodo.js

```diff
@@ -0,0 +1,55 @@
+function spCreateTodo(todo) {
+  console.log(`args: ${JSON.stringify(todo)}`);
+  const context = getContext();
+  const collection = context.getCollection();
+
+  // const response = context.getResponse();
```

Comment:
?

Reply:
Oh, this file shouild not have been included... let me take another look to see if there is anything else in there